### DISCUSSION TITLE
Making USER1 range toggle an I/O pin (turn on the VTX on Zeez F7)

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [iNavFlight/inav#6491] [Q&A]
Dear all, I wonder if you can help me understand something.

I have a Zeez F7 flight controller. In BetaFlight this has been migrated to a unified target. The VTX power on/off is controlled by the USER1 range and in the BetaFlight target definition I believe that is defined like this...

```
resource PINIO 1 B11
```
and
```
set pinio_config = 1,1,1,1
set pinio_box = 40,255,255,255
set box_user_1_name = VTX SWITCH
```

This doesn't work in iNav, probably because it hasn't been defined. How would I go about hooking up the USER1 range to an I/O pin in iNav?

**Comment by zx80r** [comment]
I'm having this exact problem on a Skystars F722HDPRO. I apologies if this is the wrong place but has there been a solution found or can someone point me to the appropriate place?

resource PINIO 1 A14
set pinio_config = 129,1,1,1
set pinio_box = 40,255,255,255

I'd like to use I nave on this Quad and don't want to install and external BEC if possible, Thanks.

**Reply by sensei-hacker** [reply]
> I'm having this exact problem on a Skystars F722HDPRO. I apologies if this is the wrong place but has there been a solution found or can someone point me to the appropriate place?
> 
> resource PINIO 1 A14 set pinio_config = 129,1,1,1 set pinio_box = 40,255,255,255
> 
> I'd like to use I nave on this Quad and don't want to install and external BEC if possible, Thanks.

To change a pin from whatever function it was assigned by the manufacturer, see this video:

https://youtu.be/2zElnakFQRU?si=f5a2r-VDRoagyz5M

**Reply by zx80r** [reply]
That's very helpful. Thank you. Ton of work though and outside my wheelhouse. I think I'll find another workaround or go back to Betaflight with this FC. Hopefully someday iNav adds resource mapping. Thanks again for this link, I'm sure if I had time I could get there with that.
